app/cdap/components/PipelineTriggers/EnabledTriggersTab/index.tsx

```diff
@@ -14,33 +14,39 @@
  * the License.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import { connect } from 'react-redux';
 import EnabledTriggerRow from 'components/PipelineTriggers/EnabledTriggersTab/EnabledTriggerRow';
-import GroupTriggerRow from 'components/PipelineTriggers/EnabledTriggersTab/EnabledGroupTriggerRow';
+import CompositeTriggerRow from 'components/PipelineTriggers/EnabledTriggersTab/EnabledCompositeTriggerRow';
 import T from 'i18n-react';
 import { useFeatureFlagDefaultFalse } from 'services/react/customHooks/useFeatureFlag';
 import PipelineTriggersTypes from 'components/PipelineTriggers/store/PipelineTriggersTypes';
 import Button from '@material-ui/core/Button';
-import { ISchedule } from 'components/PipelineTriggers/store/ScheduleTypes';
+import { ICompositeTrigger, ISchedule } from 'components/PipelineTriggers/store/ScheduleTypes';
 import {
   PipelineCount,
   PipelineListContainer,
   PipelineListHeader,
   PipelineTriggerHeader,
+  SearchTriggerTextField,
   StyledNameSpaceHeader,
   StyledPipelineNameHeader,
+  StyledTypeHeader,
   TriggersTab,
 } from 'components/PipelineTriggers/shared.styles';
 import styled from 'styled-components';
+import { IconButton, InputAdornment } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
 
 const TRIGGER_PREFIX = 'features.PipelineTriggers';
 const PREFIX = `${TRIGGER_PREFIX}.EnabledTriggers`;
 
 const AddNewTriggerButton = styled(Button)`
   background-color: #5a84e4;
-  text-transform: none;
   margin: 20px;
+  position: absolute;
+  right: 20px;
+  text-transform: none;
```

Comment:
We'd like to move to standard Mui buttons. Can this `text-transform` be removed?

Reply:
yep, removed them